platform/features/table/src/Table.js

```diff
@@ -0,0 +1,163 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define,moment*/
+
+define(
+    [
+        './DomainColumn',
+        './RangeColumn',
+        './NameColumn'
+    ],
+    function (DomainColumn, RangeColumn, NameColumn) {
+        "use strict";
+
+        /**
+         * Class that manages table metadata, state, and contents.
+         * @memberof platform/features/table
+         * @param domainObject
+         * @constructor
+         */
+        function Table(domainObject, telemetryFormatter) {
```

Comment:
I think a more clear name for this class would be `TableConfiguration`: Given a domainObject, it tells you the table configuration.  The configuration handles formatting of values, too.

Interestingly, this class knows how to load configuration from a domain object yet doesn't know how to save configuration to a domain object (the TableController handles saving).  If it's going to load configuration, it should be able to save it too-- remove that concern from the TableController.